02_configure_host.sh

```diff
@@ -111,6 +111,9 @@ if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
         else
            sudo systemctl restart network
         fi
+        if [ "${RHEL8}" = "True" ] ; then
+          sudo systemctl enable network
+        fi
```

Comment:
I'm wondering how this ends up disabled?  Was it a manual ISO install where no network was activated as part of the install?

Regardless this seems unrelated to the OS version, perhaps we should just unconditionally enable then restart it, to remove the restart duplication above?

Reply:
Ah I see the restart de-duplication is handled in https://github.com/openshift-metal3/dev-scripts/pull/690 thanks!

Reply:
I've added the enable only if the ifcfg-* files are modified

Reply:
Is the `network` service only for handling `ifcfg` scripts?  In RHEL8 that's not installed by default.  We install it so we can still use `ifcfg` scripts.  That's probably why it has to be enabled in RHEL8 and we didn't see that before.